lib/puppet.rb

```diff
@@ -211,7 +211,7 @@ def self.initialize_facts
   # code was deprecated in 2008, but this is still in heavy use.  I suppose
   # this can count as a soft deprecation for the next dev. --daniel 2011-04-12
   def self.newtype(name, options = {}, &block)
-    Puppet.deprecation_warning("Creating #{name} via Puppet.newtype is deprecated and will be removed in a future release. Use Puppet::Type.newtype instead.")
+    Puppet.deprecation_warning(_("Creating #{name} via Puppet.newtype is deprecated and will be removed in a future release. Use Puppet::Type.newtype instead."))
```

Comment:
Yeah it seems to work, I just want to make sure our translators can handle that and won't try to translate the variable name inside `#{...}` I'm shooting an email over to @LarissaLane.

Just in case anyone else is following along, the resulting pot file entry looks like:
```
msgid "\"Creating #{name} via Puppet.newtype is deprecated and will be removed in a future release. Use Puppet::Type.newtype instead.\""
msgstr ""
```